galleries/users_explain/quick_start.py

```diff
@@ -126,10 +126,14 @@
 #   b = np.matrix([[1, 2], [3, 4]])
 #   b_asarray = np.asarray(b)
 #
-# Most methods will also parse an addressable object like a *dict*, a
-# `numpy.recarray`, or a `pandas.DataFrame`.  Matplotlib allows you to
-# provide the ``data`` keyword argument and generate plots passing the
-# strings corresponding to the *x* and *y* variables.
+# Most methods will also parse a string-indexable object like a *dict*, a
+# `structured numpy array
+# <https://numpy.org/doc/stable/user/basics.rec.html#structured-arrays>`_, or a
+# `pandas.DataFrame`.  Matplotlib allows you to provide the ``data`` keyword argument
+# and generate plots passing the strings corresponding to the *x* and *y* variables.
+#
+# .. _structured numpy array: https://numpy.org/doc/stable/user/basics.rec.html#structured-arrays
```

Comment:
This link is in-lined above and causing a flake8 flag

```suggestion
```

Reply:
(alternatively we could add this file to the flake8 config ignores)

Reply:
I don't want to except the whole file from flake8. That's why I've inlined the links. Alternatively, we could use an inline comment `# noqa: E501` on the links, which we generally don't do, but seems ok in this case.